f5_openstack_agent/lbaasv2/drivers/bigip/test/test_exceptions.py

```diff
@@ -0,0 +1,521 @@
+
+
+def test_SystemUpdateException():
+    with pytest.raises(exceptions.SystemUpdateException):
+        raise getattr(exceptions, 'SystemUpdateException')()
+
+
+def test_SystemDeleteException():
+    with pytest.raises(exceptions.SystemDeleteException):
+        raise getattr(exceptions, 'SystemDeleteException')()
+
+
+def test_VirtualServerCreationException():
+    with pytest.raises(exceptions.VirtualServerCreationException):
+        raise getattr(exceptions, 'VirtualServerCreationException')()
+
+
+def test_VirtualServerQueryException():
+    with pytest.raises(exceptions.VirtualServerQueryException):
+        raise getattr(exceptions, 'VirtualServerQueryException')()
+
+
+def test_VirtualServerUpdateException():
+    with pytest.raises(exceptions.VirtualServerUpdateException):
+        raise getattr(exceptions, 'VirtualServerUpdateException')()
+
+
+def test_VirtualServerDeleteException():
+    with pytest.raises(exceptions.VirtualServerDeleteException):
+        raise getattr(exceptions, 'VirtualServerDeleteException')()
+
+
+def test_VLANCreationException():
+    with pytest.raises(exceptions.VLANCreationException):
+        raise getattr(exceptions, 'VLANCreationException')()
+
+
+def test_VLANQueryException():
+    with pytest.raises(exceptions.VLANQueryException):
+        raise getattr(exceptions, 'VLANQueryException')()
+
+
+def test_VLANUpdateException():
+    with pytest.raises(exceptions.VLANUpdateException):
+        raise getattr(exceptions, 'VLANUpdateException')()
+
+
+def test_VLANDeleteException():
+    with pytest.raises(exceptions.VLANDeleteException):
+        raise getattr(exceptions, 'VLANDeleteException')()
+
+
+def test_VXLANCreationException():
+    with pytest.raises(exceptions.VXLANCreationException):
+        raise getattr(exceptions, 'VXLANCreationException')()
+
+
+def test_VXLANQueryException():
+    with pytest.raises(exceptions.VXLANQueryException):
+        raise getattr(exceptions, 'VXLANQueryException')()
+
+
+def test_VXLANUpdateException():
+    with pytest.raises(exceptions.VXLANUpdateException):
+        raise getattr(exceptions, 'VXLANUpdateException')()
+
+
+def test_VXLANDeleteException():
+    with pytest.raises(exceptions.VXLANDeleteException):
+        raise getattr(exceptions, 'VXLANDeleteException')()
+
+
+def test_BigIPNotLicensedForVcmp():
+    with pytest.raises(exceptions.BigIPNotLicensedForVcmp):
+        raise getattr(exceptions, 'BigIPNotLicensedForVcmp')()
+
+
+@pytest.fixture
+def setup_F5MissingDependencies():
+    Options = \
+        namedtuple('Options',
+                   'errno, message, details, exception, frame')
+    set_frame = inspect.getframeinfo(inspect.currentframe())
+    items = [Options(None, None, [], None, None),
+             Options(22, 'missing', [31415926], ImportError('foodogzoo'),
+                     set_frame),
+             Options(22, 'missing', [31415926], None, set_frame)]
+    handle = Mock()
+    logging.FileHandler = Mock(return_value=handle)
```

Comment:
Mock returns a `mock.Mock()` by default so you shouldn't need to specify a`return_value`.